students/ltrisolini/Lesson03/list_lab.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+#List Lab Exercise
+
+
+#Series1
+my_list = ["Apples", "Pears", "Oranges", "Peaches"]
+
+print(my_list)
+
+user_input = input('Write another fruit: ')
+
+my_list.append(user_input)
+
+print(my_list)
+
+user_input = input('Write a number between 1-5: ')
+
+num = int(user_input)
+
+print(my_list[num - 1])
+
+my_list = ['Cherry'] + my_list
+
+print(my_list)
+
+my_list.insert(0, 'Kiwi')
+
+print(my_list)
+
+for item in my_list:
+    if item[0] == 'P':
```

Comment:
Good idea to add 'lower' to catch lower case 'p' input:  if item[0].lower() == 'p':